netkan/netkan/webhooks/github_inflate.py

```diff
@@ -80,11 +81,16 @@ def frozen_ids_from_commits(commits: List[Dict[str, Any]]) -> List[str]:
 
 
 def freeze(ids: List[str]) -> None:
-    with ModStatus.batch_write() as batch:
+    if ids:
         logging.info('Marking frozen mods...')
```

Comment:
'ids' will always be a list, this if check here is unnecessary unless we specifically want to log the generic 'Marking mods frozen' - which is probably not all that useful.

Reply:
Yeah, the idea is that freezing a mod will be pretty rare (it happens, but probably <10%), so having the log full of "Marking frozen mods..." when nothing's getting frozen is not desirable.